content/en/news/releases/1.20.x/announcing-1.20/change-notes/index.md

```diff
@@ -129,95 +139,90 @@ environment to remove the deprecated functionality.
   ([Issue #47159](https://github.com/istio/istio/issues/47159))
 
-- **Added** support for setting loggers' levels of istio-proxy in `istioctl proxy-config log` command
+- **Added** support for setting loggers' levels of istio-proxy in the `istioctl proxy-config log` command
   with `--level <level>` or `--level level=<level>`.
 
-- **Added** An analyzer for showing warning messages about incorrect/missing information related to Istio installations
-  using an External Control Plane.  ([Issue #47269](https://github.com/istio/istio/issues/47269))
+- **Added** an analyzer for showing warning messages about incorrect/missing information related to Istio installations
+  using an External Control Plane. ([Issue #47269](https://github.com/istio/istio/issues/47269))
 
-- **Added** IST0162 `GatewayPortNotDefinedOnService` message to detect the issue where Gateway port was not exposed by
-  Service.
+- **Added** IST0162 `GatewayPortNotDefinedOnService` message to detect the issue where a `Gateway` port was not exposed 
+  by `Service`.
 
-- **Fixed** `istioctl operator remove` will remove all revisions of operator controller when the revision is "default"
-  or not specified.  ([Issue #45242](https://github.com/istio/istio/issues/45242))
+- **Fixed** `istioctl operator remove` would remove all revisions of operator controller when the revision was "default"
+  or not specified. ([Issue #45242](https://github.com/istio/istio/issues/45242))
 
-- **Fixed** an issue where `verify-install` has incorrect results when installed deployments are not healthy.
+- **Fixed** an issue where `verify-install` had incorrect results when installed deployments were not healthy.
 
-- **Fixed** `istioctl experimental describe` provides wrong Gateway information when using injected gateway.
+- **Fixed** `istioctl experimental describe` provides incorrect `Gateway` information when using the injected gateway.
 
 - **Fixed** an issue where `istioctl analyze` would analyze irrelevant configmaps.
   ([Issue #46563](https://github.com/istio/istio/issues/46563))
 
-- **Fixed** `istioctl analyze` incorrectly showing an error when ServiceEntry hosts are used in a VirtualService
-  destination across namespace boundary.
+- **Fixed** `istioctl analyze` incorrectly showing an error when `ServiceEntry` hosts are used in a `VirtualService`
+  destination across a namespace boundary.
   ([Issue #46597](https://github.com/istio/istio/issues/46597))
 
-- **Fixed** an issue where `istioctl proxy-config` fails to process a config dump from file if EDS endpoints were not
+- **Fixed** an issue where `istioctl proxy-config` failed to process a config dump from a file if EDS endpoints were not
   provided.
   ([Issue #47505](https://github.com/istio/istio/issues/47505))
 
 - **Removed** `istioctl experimental revision tag` command, which was graduated to `istioctl tag`.
-
-## Documentation changes
-
-- **Fixed** add ambient test framework flag for quick running integration test.
```

Comment:
I removed this since I saw the PR was reverted, and the issue was re-opened. Please correct me if I'm wrong.